tests/unit/services/run_error_handling_tests.py

```
@@ -0,0 +1,220 @@
+                continue
+
+            # Verify source in detail
+            if "test_source" not in http_exc.detail:
+                print(f"  ✗ {test_name}: Source not in error detail")
+                tests_failed += 1
+                continue
+
+            print(f"  ✓ {test_name}")
+            tests_passed += 1
+
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)